### PR TITLE
Update R&D organization and AORs

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Updated to match the new org structure and AORs.

### CODE REVIEW

**Review by justinegeffen — DISMISSED** [review]
Thanks for this update!

**Review by kwiersgalla — DISMISSED** [review]
Thanks Joram.

File: operations/research-and-development/organization/README.md
Comment (DSchalla) [inline operations/research-and-development/organization/README.md:54]:
```suggestion
* **[Governance, Risk, and Compliance (GRC)](/operations/research-and-development/organization/grc.md)**
* **[Product Security](/operations/research-and-development/organization/product_security.md)**
```

File: operations/research-and-development/organization/README.md
Comment (DSchalla) [inline operations/research-and-development/organization/README.md]:
```suggestion
The Security division is responsible for the implementation and monitoring of the company's security program.
```

File: SUMMARY.md
Comment (DSchalla) [inline SUMMARY.md:62]:
```suggestion
    * [GRC](/operations/research-and-development/organization/grc.md)
    * [Product Security](/operations/research-and-development/organization/product_security.md)
```

**Review by DSchalla — CHANGES_REQUESTED** [review]
Looks good overall, thanks Joram! I've committed a team page for the GRC team to the branch directly and made some minor change requests to add them in all places.

**Comment by DSchalla** [comment]
@justinegeffen @kwiersgalla Apologies, it seems my commit addition invalidated your review.

**Review by justinegeffen — DISMISSED** [review]

**Comment by justinegeffen** [comment]
> @justinegeffen @kwiersgalla Apologies, it seems my commit addition invalidated your review.

No worries! I've requested another review from you - I think if you approve it'll mark your requests as done. I never know with GitHub, but your changes have been merged!

**Review by justinegeffen — DISMISSED** [review]

**Review by DSchalla — DISMISSED** [review]
Looks good, thanks @justinegeffen!

**Review by kwiersgalla — DISMISSED** [review]

**Review by marianunez — DISMISSED** [review]
Thanks @jwilander! I made some minor updates on Growth and Self-Serve AORs. My changes dismissed the existing approvals 😬

File: operations/research-and-development/organization/growth.md
Comment (catalintomai) [inline operations/research-and-development/organization/growth.md]: Hasn't this been removed?

File: operations/research-and-development/organization/growth.md
Comment (catalintomai) [inline operations/research-and-development/organization/growth.md]: Nuturing?

File: operations/research-and-development/organization/growth.md
Comment (jwilander) [reply operations/research-and-development/organization/growth.md]: @marianunez ?

File: operations/research-and-development/organization/growth.md
Comment (jwilander) [reply operations/research-and-development/organization/growth.md]: @marianunez ?

Comment (marianunez) [reply operations/research-and-development/organization/growth.md]: Typo, it is `Nurturing`

File: operations/research-and-development/organization/growth.md
Comment (marianunez) [reply operations/research-and-development/organization/growth.md]: Yes, the original Admin Advisor was disabled. The concept of suggesting specific features depending on usage is still discussed but I'll remove this and the new features are considered under upsells that Self-Serve team owns.

**Review by saturninoabril — APPROVED** [review]
Thanks Joram, LGTM for QA AOR.

File: operations/research-and-development/organization/README.md
Comment (Willyfrog) [inline operations/research-and-development/organization/README.md:13]: Maybe this sounds better once the teams are no longer named Core, but right now it feels like trying to define  a word with itself.
But I don't have a better suggestion for it. :/

**Review by Willyfrog — APPROVED** [review]

File: operations/research-and-development/organization/README.md
Comment (jwilander) [reply operations/research-and-development/organization/README.md:13]: Yeah, I struggled with that a bit. "The core team builds the core" is a little unnecessary but it also felt weird not providing any description.

**Review by m1lt0n — APPROVED** [review]

**Comment by jwilander** [comment]
Going to merge as is in order to get it in. If folks have updates they'd like to make please follow-up with another PR.

**Pull request merged**
